iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/digitaltwin/DigitalTwinServiceClientTests.java

```diff
@@ -1,38 +1,90 @@
 package tests.integration.com.microsoft.azure.sdk.iot.digitaltwin;
 
-import com.microsoft.azure.sdk.iot.device.exceptions.TransportException;
+import com.microsoft.azure.sdk.iot.device.IotHubClientProtocol;
+import com.microsoft.azure.sdk.iot.service.digitaltwin.DigitalTwinAsyncClient;
+import com.microsoft.azure.sdk.iot.service.digitaltwin.DigitalTwinClient;
+import com.microsoft.azure.sdk.iot.service.digitaltwin.models.BasicDigitalTwin;
+import com.microsoft.azure.sdk.iot.service.exceptions.IotHubException;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.Timeout;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import tests.integration.com.microsoft.azure.sdk.iot.digitaltwin.helpers.E2ETestConstants;
+import tests.integration.com.microsoft.azure.sdk.iot.digitaltwin.simulator.TestDigitalTwinDevice;
+import tests.integration.com.microsoft.azure.sdk.iot.helpers.Tools;
 import tests.integration.com.microsoft.azure.sdk.iot.helpers.annotations.DigitalTwinTest;
 
 import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.Collection;
+import java.util.UUID;
+
+import static com.microsoft.azure.sdk.iot.device.IotHubClientProtocol.MQTT;
+import static com.microsoft.azure.sdk.iot.device.IotHubClientProtocol.MQTT_WS;
+import static java.util.Arrays.asList;
+import static org.junit.Assert.assertEquals;
 
 @DigitalTwinTest
+@Slf4j
+@RunWith(Parameterized.class)
 public class DigitalTwinServiceClientTests {
 
+    protected static String iotHubConnectionString = "";
+    private TestDigitalTwinDevice testDevice;
+    private DigitalTwinClient digitalTwinClient = null;
+    private static final String DEVICE_ID_PREFIX = "DigitalTwinE2ETests_";
```

Comment:
nit: change prefix to DigitalTwinClientE2ETests, or DigitalTwinServiceClientTests_ to match file name